qgepplugin/processing_provider/provider.py

```diff
@@ -88,7 +88,7 @@ def load(self):
         ProcessingConfig.addSetting(Setting(self.name(),
                                             'SWMM_PATH',
                                             self.tr("SWMM executable"),
```

Comment:
```suggestion
                                            self.tr(r'SWMM executable (e.g. C:\Program Files (x86)\EPA SWMM 5.1.013\swmm55.exe)'),
```

How about this to help people with the configuration?

Reply:
Seems clever